reference/configuration/doctrine.rst

```diff
@@ -128,6 +130,17 @@ The following block shows all possible configuration keys:
     yet, you may get ``PDOException`` errors because Doctrine will try to
     guess the database server version automatically and none is available.
 
+.. note::
```

Comment:
ci complained about `.. versionadded:: 2.3`. 

i just do a note as it seems we don't want versionadded for 3rd party components.